package.json

```diff
@@ -62,7 +62,7 @@
     "babel-preset-flow-vue": "^1.0.0",
     "browserstack-local": "^1.4.0",
     "buble": "^0.19.8",
-    "chromedriver": "^79.0.0",
+    "chromedriver": "^81.0.0",
```

Comment:
Let's keep this change out of the PR. If needed I will upgrade this on dev and you can then merge or rebase against it

Reply:
I did this change to be able to run e2e tests. Without this update, e2e tests can't run: https://circleci.com/gh/vuejs/vue-router/5121 😉

Reply:
oh wow, they are still green, that's so wrong lol
I updated on `dev` the chromedriver but there are a few tests failing I need to fix. After that you can merge dev into this (don't worry about the git commit history, we squash at the end)

Reply:
It's good now!